pkg/errors/errors.go

```diff
@@ -277,3 +278,9 @@ func CheckError(err error) {
 		os.Exit(1)
 	}
 }
+
+// UsageError checks command usage error.
+func UsageError(cmd *cobra.Command, format string, args ...interface{}) error {
+	msg := fmt.Sprintf(format, args...)
```

Comment:
it would seem reasonable to me for UsageError to just take the msg string, and expect callers to have already invoked fmt.Sprintf before passing the message in.